defender/utils.py

```diff
@@ -202,8 +202,8 @@ def unblock_username(username, pipe=None):
         pipe = REDIS_SERVER.pipeline()
         do_commit = True
     if username:
-        pipe.delete(get_username_attempt_cache_key(username))
-        pipe.delete(get_username_blocked_cache_key(username))
+        pipe.delete(get_username_attempt_cache_key(username.lower()))
+        pipe.delete(get_username_blocked_cache_key(username.lower()))
```

Comment:
Sorry, I didn't see this sooner. I think it makes sense to always store usernames in the same case, lower probably. 

We could so add the .lower() inside of the ``get_username_attempt_cache_key()`` and ``get_username_blocked_cache_key()`` this way we don't have to always pass in a lower case username, and it keep it all in one place.